### PR TITLE
sendMessageBirdWhatsappMsg(): Add Kotlin implementation

### PR DESCRIPTION
**Pull request by daniel-shuy**
Partially implements appwrite/appwrite#1891

Tested using [MessageBird WhatsApp Sandbox](https://dashboard.messagebird.com/en/whatsapp/sandbox) (see https://support.messagebird.com/hc/en-us/articles/360002109957-Start-testing-with-the-MessageBird-WhatsApp-Sandbox)

# Input
image

# Successful Execution
image

# Output
image
- The Function Data is printed to the console
- The MessageBird response is printed to the console

image

A few ways this improves over other Kotlin examples:
- Uses the [Gradle Shadow Plugin](https://imperceptiblethoughts.com/shadow/) to generate a uber/fat-JAR, and to minimize the JAR size (without it, just including the Kotlin standard library, [Appwrite SDK for Kotlin](https://github.com/appwrite/sdk-for-kotlin) and [CloudConvert Java SDK](https://github.com/cloudconvert/cloudconvert-java) would already exceed the 10MB size limit)
- Uses the [Gradle Application Plugin](https://docs.gradle.org/current/userguide/application_plugin.html) to build and create a TAR file in a single step (using `gradle shadowDistTar`), removing the need to manually TAR the JAR file
- Validates enum values and displays possible options in error message

### CODE REVIEW

**Review by Meldiron — APPROVED** [review]
Looks good to me

File: kotlin/send-message-bird-whatsapp-msg/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```
Comment (christyjacob4) [inline kotlin/send-message-bird-whatsapp-msg/gradle/wrapper/gradle-wrapper.properties:1]: You can remove this file

File: kotlin/send-message-bird-whatsapp-msg/gradlew [hunk 2]
```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
+
+#
+# Copyright 2015 the original author or authors.
```
Comment (christyjacob4) [inline kotlin/send-message-bird-whatsapp-msg/gradlew:4]: You can remove this file

File: kotlin/send-message-bird-whatsapp-msg/gradlew.bat [hunk 3]
```diff
@@ -0,0 +1,89 @@
+@rem
+@rem Copyright 2015 the original author or authors.
+@rem
```
Comment (christyjacob4) [inline kotlin/send-message-bird-whatsapp-msg/gradlew.bat:3]: You can remove this file

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
You can remove the `gradle-wrapper` files to stay consistent with the other demos

Please use this as a reference https://github.com/appwrite/demos-for-functions/tree/master/kotlin/get-covid-stats

**Comment by daniel-shuy** [comment]
@christyjacob4 done. Seems like an odd decision though since including the Gradle Wrapper seems like a standard practice for most Gradle projects

**Comment by christyjacob4** [comment]
@daniel-shuy I agree, gradle-wrapper would be handy if we were running this locally. 

But since we are creating a jar file out of this, a gradle-wrapper would be irrelevant. 🙂

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+is%3Aissue+label%3Ahacktoberfest+created%3A%3E2021-01-01+is%3Aopen&type=issues) and help make Appwrite better :)

**Pull request merged**